packages/viewer/src/store/modules/position/actions/setCenter.ts

```diff
@@ -13,8 +13,26 @@ import useGeolocationStore from '@/store/modules/geolocation'
 export default function setCenter(
     this: PositionStore,
     center: SingleCoordinate,
+    preserveGeolocationTracking: boolean,
     dispatcher: ActionDispatcher
+): void
+export default function setCenter(
+    this: PositionStore,
+    center: SingleCoordinate,
+    dispatcher: ActionDispatcher
+): void
+export default function setCenter(
+    this: PositionStore,
+    center: SingleCoordinate,
+    preserveGeolocationTrackingOrDispatcher: boolean | ActionDispatcher,
+    dispatcherOrNothing?: ActionDispatcher
```

Comment:
Can you switch to using an "options" bag, instead of having the boolean straight into the signature?

I think it would clarify a bit better the code when the function is used